app/vep/utils/web_metadata.py

```diff
@@ -24,3 +24,4 @@ def get_vep_support_location(genome_id):
         raise HTTPError
     except Exception as e:
         logging.error(e)
+        raise Exception
```

Comment:
This was flagged by Pylance: without re-raising, it will make this funciton to return `None`, which is not expected downstream, resulting in another, different Exception ([at this line](https://github.com/Ensembl/ensembl-web-tools-api/blob/994dc962d07672df2b5a3578e4b873159fe09f79/app/vep/models/pipeline_model.py#L69)).